### PR TITLE
Added token type for numbers with trailing letters

### PR DESCRIPTION
**Pull request by sveinbjornt**

### CODE REVIEW

File: src/tokenizer/tokenizer.py [hunk 1]
```diff
@@ -648,6 +654,13 @@ def parse_digits(w):
         sec = int(p[2])
         if (0 <= h < 24) and (0 <= m < 60) and (0 <= sec < 60):
             return TOK.Time(w, h, m, sec), s.end()
+    s = re.match(r'\d+[A-Za-z]{1}', w)
```
Comment (vthorsteinsson) [inline src/tokenizer/tokenizer.py]: I suggest moving this to after the time and date checks (HH:MM and YYYY/MM/DD and derivatives), considering those as more specific and this as slightly less specific. Also, maybe the regex should end with something like `(?![:alpha:])` to prevent "10million" or "1000isk" from matching. We only want a number plus one character, not multiple chars.

File: src/tokenizer/tokenizer.py [hunk 2]
```diff
@@ -648,6 +654,13 @@ def parse_digits(w):
...
+    if s:
```
Comment (vthorsteinsson) [inline src/tokenizer/tokenizer.py]: I suggest adding an `int(n)` in the NumberWithCharacter() constructor, i.e. storing the number as an integer, not as a string. Another possibility would simply be not to break down the string into two parts, but just to store it as-is. The choice depends on what is most useful for downstream processing of addresses.

**Comment by vthorsteinsson** [comment]
...and it would be useful to add tests for this, i.e. in `test/test_tokenizer.py`, checking the corner cases: "Laugavegur 100", "Laugavegur 100a", "Laugavegur 200þ", "Laugavegur 200milljónir", etc.

**Comment by sveinbjornt** [comment]
OK, made some fixes and worked around the units of measurement issue we discussed.
